src/compiler/emitter.ts

```diff
@@ -133,14 +133,14 @@ module ts {
         };
     }
 
-    function getLineOfLocalPosition(currentSourceFile: SourceFile, pos: number) {
+    function getBasedLineOfLocalPosition(currentSourceFile: SourceFile, pos: number) {

```

Comment:
Bad edit.  I changed "getZeroBasedLine..."  to "getBasedLine..."   instead of "getLine..."